network-api/networkapi/wagtailpages/templates/buyersguide/product_page.html

```diff
@@ -96,175 +96,182 @@ <h2 class="h3-heading">{% trans "What could happen if something goes wrong" %}</
+              <span class="tw-py-2 tw-px-7 medium:tw-flex-1 tw-flex-shrink-0 tw-relative medium:tw-w-auto  tw-w-[300px] small:tw-w-[200px] tw-opacity-0 small:tw-hidden">mobile</span>
+              <span
+                data-product-label="0"
+                class="{{tab_class}} tw-w-[200px]"
+              >
+                {% trans "Privacy" %}
+                {% if product.data_collection_policy_is_bad or product.data_control_policy_is_bad or product.track_record_is_bad %}
+                  <img class="tw-h-6 tw-w-6 tw-ml-3" src="{% static "_images/buyers-guide/warning-ding.svg" %}" alt="warning">
+                {% endif %}
+              </span>
+              <span
+                data-product-label="1"
+                class="{{tab_class}} tw-w-[200px]"
+              >
+                {% trans "Security" %}
+                {% if product.show_ding_for_minimum_security_standards %}
+                  <img class="tw-h-6 tw-w-6 tw-ml-3" src="{% static "_images/buyers-guide/warning-ding.svg" %}" alt="warning">
+                {% endif %}
+              </span>
+              <span
+                data-product-label="2"
+                class="{{tab_class}} tw-w-[200px]"
+              >
+                {% trans "A.I." %}
+              </span>
             </div>
-
-            {% if product.signup_requirement_explanation %}
-            <p class="pni-product-helptext mb-0 mt-3">{{product.signup_requirement_explanation}}</p>
-            {% endif %}
-          </section>
-
-
-          <section class="criterion">
-            <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7">{% trans "What data does it collect?" %}</h3>
-
-            <div class="row">
-              {% trans "Personal"      as personal_data_collected %}
-              {% trans "Biometric"     as biometric_data_collected %}
-              {% trans "Social"        as social_data_collected %}
-
-              <div class="col-12 col-md-4">
-                {% include "fragments/signup_requirement.html" with title=personal_data_collected   value=product.personal_data_collected    %}
-              </div>
-              <div class="col-12 col-md-4">
-                {% include "fragments/signup_requirement.html" with title=biometric_data_collected  value=product.biometric_data_collected   %}
-              </div>
-              <div class="col-12 col-md-4">
-                {% include "fragments/signup_requirement.html" with title=social_data_collected     value=product.social_data_collected      %}
+            <div class="tw-overflow-hidden">
+              <div id="product-tab-content" class="tw-w-full tw-flex tw-transition-transform tw-ease-in-out tw-duration-300">
+                <section class="tw-w-full tw-flex-shrink-0" data-product-label="0">
+                  {% if product.product_type == 'general' %}
+                    <div class="criterion">
```

Comment:
Semantic HTML question: why did this become a `div` instead of staying a `section`? It feels like it would be more semantic for `tw-w-full tw-flex-shrink-0` to be a div, and the sections to stay sections.